webapp/loggedin/modules/users/userInvite/userInviteView.scss

```diff
@@ -0,0 +1,24 @@
+.user-invite-view {
```

Comment:
I would call this just ```.user-invite```

Reply:
ok

Reply:
should we remove the suffix -view even in userView?

Reply:
I think so